Teensy/src/gpio/gpio.cpp

```diff
@@ -99,7 +105,29 @@ static daq_EngineRev_t gpio_getEngineRevs()
 
 static DamperPos_S gpio_getDamperPosition()
 {
+    /*
+    Read and convert sensor value potentiometer (which should be a value from 0 to 1023 
+    based on the voltage with 3.3v max) then use the lambda expression below to calculate
+    the mm value for the potentiometer using the sensor value, make sure the max value 
+    for the sensor is 1023 (when the potentiometer is fully closed) and not anything below.
+    */ 
+    auto equation = [](int32_t x) {
```

Comment:
I would prefer not to use auto. You should be able to use int32. No other comments really though! Looks good. Let me know when you'll be able to test the code